rbs/prism/SignatureTranslatorPrism.cc

```diff
@@ -0,0 +1,125 @@
+#include "rbs/prism/SignatureTranslatorPrism.h"
+#include "core/errors/rewriter.h"
+#include "rbs/prism/MethodTypeToParserNodePrism.h"
+#include "rbs/prism/TypeParamsToParserNodesPrism.h"
+#include "rbs/prism/TypeToParserNodePrism.h"
+#include "rbs/rbs_common.h"
+
+using namespace std;
+using namespace sorbet::parser::Prism;
+
+namespace sorbet::rbs {
+
+pm_node_t *
+SignatureTranslatorPrism::translateAssertionType(absl::Span<pair<core::LocOffsets, core::NameRef>> typeParams,
+                                                 const rbs::RBSDeclaration &assertion) {
+    rbs_string_t rbsString = makeRBSString(assertion.string);
+    const rbs_encoding_t *encoding = RBS_ENCODING_UTF_8_ENTRY;
+
+    Parser parser(rbsString, encoding);
```

Comment:
Uniform initialization

```suggestion
    Parser parser{rbsString, encoding};
```